code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm

```diff
@@ -5,14 +5,14 @@
 	name = "Aasimar"
 	id = "aasimar"
 	desc = "<b>Aasimar</b><br>\
-	Aasimar are born of a rare union between Humans and Angels. \
-	They bear the mark of their celestial touch through many varying physical features. \
-	Their looks resemble the traditional characteristics of whichever of the Gods the Angel parent was associated with. \
-	Most commonly, Aasimar are similar to Humans, albeit taller, and possess uncanny beauty. \
-	They have strangely colored skin and are more physically frail than the average Human. \
-	Because of their upbringing, they make for natural conduits for godly powers. \
-	Rockhills populace holds them with a mixture of uneasy fear or, and respect. \
-	It is also widely believed that an Aasimars death is a bad omen..."
+	Aasimar are born of a rare union between mortals and Celestials of the Ten. \
+    They bear the mark of their celestial touch through many varying physical features. \
+    Their looks resemble the traditional characteristics of whichever of the Gods the Celestial parent was associated with. \
```

Comment:
Celestials from the Marvel Cinematic Universe?
Was this even approved by a lore maintainer?

Reply:
Ask Centurion, he provided the description.